config/initializers/geocoder_mapbox.rb

```diff
@@ -0,0 +1,24 @@
+# Mapbox relevance is defined from 0 to 1, where 0 is the least relevant and 1 is the most relevant
+# @see https://docs.mapbox.com/api/search/geocoding/#geocoding-response-object
+RELEVANCE_LIMIT = 1
```

Comment:
Don't we need to set it to something < 1?

Reply:
Let me put 0.9. I'm not really sure, I just got a lot of relevance = 1, even when it was weird addresses.